src/Framework/Core/Form/Control/Link/__tests__/Link.component-test.js

```diff
@@ -29,4 +31,24 @@ describe('components <LinkComponent />', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+  it('linkTo function works correctly', () => {
+    const props = {
+      label: messages.title,
+      route: mockRoute,
+    };
+    const wrapper = shallow(
+      <LinkComponent {...props} />
+    );
+    const instance = wrapper.instance();
+    const linkToSpy = sinon.spy(instance, 'linkTo');
+    instance.forceUpdate();
+    wrapper.find(TouchableHighlight).props().onPress();
+    sinon.assert.calledOnce(linkToSpy);
+
+    const stubLinking = sinon.stub(Linking, 'canOpenURL');
+    stubLinking.withArgs(mockRoute).resolves(true);
+    wrapper.find(TouchableHighlight).props().onPress();
+    stubLinking.restore();
```

Comment:
restore is not necessary since stub is only called once

Reply:
Ok ! Thanks.